core/nodepool/config/templates/cloud-config-worker

```diff
@@ -481,7 +481,7 @@ coreos:
         --container-runtime={{.ContainerRuntime}} \
         --rkt-path=/usr/bin/rkt \
         --rkt-stage1-image=coreos.com/rkt/stage1-coreos \
-        --node-labels=kubernetes.io/role=node{{if .NodeLabels.Enabled}},{{.NodeLabels.String}}{{end}} \
+        --node-labels=kubernetes.io/role={{ toLabel .NodePoolName }}{{if .NodeLabels.Enabled}},{{.NodeLabels.String}}{{end}} \
```

Comment:
this label doesn't match selector in controller above

Reply:
Not sure what you are highlighting here - yes its a different label to the one used to identify masters.